DESCRIPTION

```diff
@@ -46,7 +46,6 @@ Suggests:
     xml2
 Remotes:
     r-lib/downlit,
-    rstudio/bookdown,
     rstudio/bslib,
```

Comment:
Do we still need bslib? And dev downlit?

Reply:
I think not, took those out.